src/components/App.jsx

```diff
@@ -7,10 +7,10 @@ import classNames from 'classnames'
 import Sidebar from './Sidebar'
 import Alerter from './Alerter'
 
-const App = ({ t, children }) => (
+export const App = ({ t, children }) => (
   <div className={classNames(styles['pho-wrapper'], styles['coz-sticky'])}>
 
-    <Sidebar />
+    <Sidebar t={t} />
```

Comment:
@goldoraf I agree but even with the unwrapped component, it broke the tests. We can export the unwrapped component, but if it used a translated component, the test will break. And for now, with Jest, I didn't see a way to do shallow rendering (render the first level of components).